index.mjs.js

```diff
@@ -1,16 +1,10 @@
-import {error, invalidArgument} from './src/internal/throw';
-
-if(typeof Object.create !== 'function') error('Please polyfill Object.create to use Fluture');
-if(typeof Object.assign !== 'function') error('Please polyfill Object.assign to use Fluture');
-if(typeof Array.isArray !== 'function') error('Please polyfill Array.isArray to use Fluture');
```

Comment:
This chunk was useless, because `rollup` changes the order of code evaluation in such a way that this code is reached only after some parts have already tried to use these functions. It's also not super important, as it only served to slightly improve the error messages.